pkg/cli/admin/release/new.go

```diff
@@ -1392,7 +1392,7 @@ func writePayload(w io.Writer, is *imageapi.ImageStream, cm *CincinnatiMetadata,
 
 	// read each directory, processing the manifests in order and updating the contents into the tar output
 	if err := iterateExtractedManifests(ordered, metadata, func(directory string, contents []os.FileInfo, operator string) error {
-		transform := fallible(NopManifestMapper)
+		transform := NewSimpleVersionsMapper(is.Name)
```

Comment:
Could we use `NewComponentVersionsMapper(is.Name, nil, nil)` directly here?

I am wondering if this PR alone would fix OCPBUGS-19824 without the [NO-JIRA: adm release](https://github.com/openshift/oc/pulls?q=is%3Apr+author%3Apetr-muller+is%3Aclosed+NO-JIRA%3A+adm+release) pulls from you recently?

My guess: yes it would.

Reply:
Understood. Thanks.